bumble/pairing.py

```diff
@@ -222,6 +223,14 @@ async def key_distribution_response(
             ),
         )
 
+    async def generate_passkey(self) -> int:
+        """
+        Return a passkey value between 0 and 999999 (inclusive).
+        """
+
+        # By default, generate a random passkey.
+        return secrets.randbelow(1000000)
```

Comment:
Unrelated: Currently, both PairingConfig and PairingDelegate don't have information about the pairing session. I think we may need to add such information like a connection attribute in the future.